rt_bi_emulator:
  kn2:
    ros__parameters:
      render: True
      id: "https://rezateshnizi.com/tower_bridge/defintion/king"
      updateInterval: 0.25
      timesSecs:
        - 0.00
        - 15.00
        - 18.00
        - 35.00
        - 51.00
        - 90.00
      cutOffTime: inf
      centersOfRotation:
        - "[17.500, 248.25]"
        - "[177.75, 247.25]"
        - "[178.50, 197.50]"
        - "[178.50, 197.50]"
        - "[179.00, 32.500]"
        - "[500.00, 32.000]"
      polygons:
        - "[[15.000, 245.75], [20.000, 245.75], [20.000, 250.75], [15.000, 250.75]]"
        - "[[175.25, 244.75], [180.25, 244.75], [180.25, 249.75], [175.25, 249.75]]"
        - "[[176.00, 195.00], [181.00, 195.00], [181.00, 200.00], [176.00, 200.00]]"
        - "[[176.00, 195.00], [181.00, 195.00], [181.00, 200.00], [176.00, 200.00]]"
        - "[[176.50, 30.000], [181.50, 30.000], [181.50, 35.000], [176.50, 35.000]]"
        - "[[495.00, 29.500], [500.00, 29.500], [500.00, 35.500], [495.00, 35.500]]"
